runners/katacoda/templates/executeFile.md

```diff
@@ -0,0 +1,9 @@
+<%= text; %>
+
+<%= cdCommand; %>
+
+Run <%= name; %> with this bash-command.  
+
+`bash <%= name; %>`{{execute T1}}
```

Comment:
It could be that the executed files start a server. Then we need to adjust the terminal handling.
We can pass this as an optional parameter (e.g. 'asynchronous: true'). And in some cases we have to pass command line arguments. For this we need also some additional paramter.